src/algokit/core/tasks/ipfs.py

```diff
@@ -115,31 +122,34 @@ def upload_to_web3_storage(
         logger.debug(f"Timeout set to {timeout} seconds based on {num_chunks} chunks.")
 
     headers = {
-        "Authorization": f"Bearer {api_key}",
-        "Content-Type": "application/x-www-form-urlencoded",
-        "X-Name": name or file_path.name,
+        "accept": "application/json",
+        "Authorization": f"Bearer {jwt}",
     }
 
+    pinata_options = {"cidVersion": "1"}
+    data = {"pinataOptions": json.dumps(pinata_options)}
+    files = {"file": (name or file_path.name, file_content)}
     try:
         response = httpx.post(
-            "https://api.web3.storage/upload",
+            url="https://api.pinata.cloud/pinning/pinFileToIPFS",
+            data=data,
+            files=files,
             headers=headers,
-            timeout=timeout,
-            content=content or file_content,
+            timeout=DEFAULT_TIMEOUT,
         )
         response.raise_for_status()
-        cid = response.json().get("cid")
-        if not isinstance(cid, str):
-            raise ValueError("CID is not a string.")
-        return cid
+        ipfs_hash = response.json().get("IpfsHash")
```

Comment:
You can keep the variable named cid if you like. Cid is a hash of the file contents.